.circleci/config.yml

```diff
@@ -170,5 +236,7 @@ workflows:
         jobs:
             - "python-3.6"
             - "node"
-            - "test"
+            - "server-test"
+            - "standalone-test"
+            - "unit-test"
```

Comment:
These were taking way too long. Breaking up the server, standalone and unit tests.